sciLifeLab_utils/run_denovo.py

```diff
@@ -104,6 +104,17 @@ def qc_report(state, **kwargs):
     from . import run_QC_report
     run_QC_report.main(_to_namespace(state.params, **kwargs))
 
+@cli.command()
+@common_options
+@click.option('--source', required=True, type =click.Path(exists=True), help= "Path to QC analysis folder")
+@click.option('--uppnexid', required=True, type = str, help ="Destination Uppnex id")
+@pass_state
+def delivery(state, **kwargs):
```

Comment:
personal taste: I am not a big fun of **kwargs as makes more difficult the debugging. I prefer to see all the parameters of a function specified rather than having **kwargs (but this is my personal taste)